torch/_inductor/mkldnn_lowerings.py

```diff
@@ -162,11 +163,55 @@ def convolution_binary_inplace(
                 )
             )
 
+        def mlp_linear_silu_linear_mul(
+            x: TensorBox,
+            w: List[TensorBox],
+            b: List[TensorBox],
+            attr,
+            scalars,
+            algorithm,
+            layout=None,
+        ):
+            assert use_max_autotune()
+            b = [
+                bias if bias is None else ir.ExternKernel.realize_input(bias)
+                for bias in b
+            ]
+
+            choices: List[ChoiceCaller] = []
+            transposed_w0 = permute(w[0], [1, 0])
+            *_, layout, x, transposed_w0 = mm_args(x, transposed_w0, layout=layout)
+
+            if use_cpp_packed_gemm_template(layout, x, transposed_w0):
+                kwargs = dict(
+                    has_bias=[bias is not None for bias in b],
+                    trans_w=True,
+                )
+
+                input_nodes = [x, w[0], w[1]]
+                input_nodes.extend([bias for bias in b if bias is not None])
+
+                CppPackedMLPTemplate.add_choices(
+                    choices,
+                    layout,
+                    input_nodes,
+                    **kwargs,  # type: ignore[arg-type]
+                )
+
+                assert len(choices) != 0
+                result = autotune_select_algorithm(
+                    "linear_unary",
+                    choices,
+                    input_nodes,
+                    layout,
+                )
+                return result
```

Comment:
Not all paths return the value?

Reply:
Thanks and add the assert since there is no aten fallback kernel.